src/validators.rs

```diff
@@ -41,6 +42,34 @@ pub struct StakeStats {
     pub count: u64,
 }
 
+/// Reward for validator
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct Reward {
+    /// The owner address is the base58 check-encoded public key of
+    /// the owner's wallet address.
+    pub account: String,
+    /// The reward amount.
+    pub amount: i64,
```

Comment:
You could use the Hnt type from this repo:
```
    #[serde(deserialize_with = "Hnt::deserialize")]
    pub amount: Hnt,
 ```

Reply:
done.